examples/imgui_impl_vulkan.cpp

```diff
@@ -721,7 +721,12 @@ bool ImGui_ImplVulkan_CreateDeviceObjects()
 
     VkPipelineMultisampleStateCreateInfo ms_info = {};
     ms_info.sType = VK_STRUCTURE_TYPE_PIPELINE_MULTISAMPLE_STATE_CREATE_INFO;
-    ms_info.rasterizationSamples = VK_SAMPLE_COUNT_1_BIT;
+    if (v->MSAASamples != 0) {
+        ms_info.rasterizationSamples = v->MSAASamples;
+    }
+    else {
+        ms_info.rasterizationSamples = VK_SAMPLE_COUNT_1_BIT;
+    }
```

Comment:
Thanks!
Might want to remove those braces, update local changelog at the top of imgui_impl_vulkan.cpp and docs/Changelog.
I'll do those changes if you need.

Reply:
Oops, I'll remove those braces. I didn't notice that it wasn't in line with ImGui style. Sorry!

Would text like this be ok for the changelog in imgui_impl_vulkan.cpp?
```
//  2019-07-31: Vulkan: Added support for specifying multisample count. Set ImGui_ImplVulkan_InitInfo::MSAASamples to one of the VkSampleCountFlagBits values to use, default is non-multisampled as before.
```
And would it be the same text, in the version 1.73 section, for docs/Changelog?

Reply:
Yes that's good, thanks!

Reply:
(I'm actually doing a bit of a rollback and releasing a hotfix 1.72b to fix a navigation issue now, so your patch may be in it or not.)

Reply:
OK great, I've added those changes to the PR. Thanks!